config/manifest-helpers.js

```diff
@@ -25,17 +29,16 @@ function getWebpackEntryPoints(manifests) {
 
 function displayApplications() {
   const root = path.join(__dirname, '../..');
-  getAppManifests(root)
-    .forEach(manifest => {
-      console.log(manifest.appName);
-      console.log(path.relative(root, manifest.filePath));
-      console.log(`Production: ${manifest.production || false}`);
-      console.log('');
-    });
+  getAppManifests(root).forEach(manifest => {
```

Comment:
Much clearer, does it enforce new lines for chained calls but not for individual calls? If so, that's a nice distinction.

Reply:
I'm sure someone smarter than me could explain how it determines were to wrap lines and where to keep statements on the same line :( I just like not having to think about it.